.github/repository-settings.md

```diff
@@ -0,0 +1,35 @@
+# Repository settings
+
+This document describes any changes that have been made to the
+settings for this repository beyond the [OpenTelemetry default repository
+settings](https://github.com/open-telemetry/community/blob/main/docs/how-to-configure-new-repository.md#repository-settings).
+
+## General
+
+No changes
+
+## Collaborators and Teams
+
+* There is currently no `javascript-triagers` role
+* `javascript-maintainers` has `Admin` permission
+
+## Branches
+
+## Branch protection rules
+
+### `dependabot/**/**`
+
+There is currently not an explicit rule for this branch pattern.
+Our dependencies are managed by a bot which creates PRs from a fork.
+
+### `gh-pages`
+
+This is a special branch which we use to publish the automatically generated docs.
+It is exempt from most protections.
+ 
+* Allow force pushes from everyone
```

Comment:
```suggestion
* Allow force pushes from everyone with the write permission
```

Reply:
This is a direct copy of the text from the configuration UI. I think write permission is implied in this instance.